.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    open-pull-requests-limit: 1000
```

Comment:
Without this it defaults to some very small limit. I'm not sure why you wouldn't just want to see all at once?

Reply:
Ya that is a weird default. I guess it can get "noisy" but whatever that does not seem like the proper way to mitigate that